Resources/Help/analyses/ldbernoulli_nl.md

```diff
@@ -1,54 +1,46 @@
+- Samenvatting: Geeft een tabel met de kengetallen van de geselecteerde variabele weer.
 
-### Plots
-- Barplot: toon een barplot van de geselecteerde variabele.
-- Empirische cumulatieve verdeling: geeft een empirische cumulatieve verdelingsgrafiek van de geselecteerde variabele weer.
+### Grafieken
+- Staafdiagram: Toont een staafdiagram van de geselecteerde variabele.
 
-## Parameters schatten
-- Maximale waarschijnlijkheid: pas de theoretische verdeling aan de geselecteerde variabele aan door de waarschijnlijkheidsfunctie te maximaliseren. De parameterwaarden die zijn gespecificeerd in het gedeelte Verdeling tonen dienen als startpunt voor de procedure.
+## Schat Parameters
+- Meest aannemelijke schatter: Schat de parameters met de waarden in het domein waar de aannemelijksheidsfunctie maximaal is. De aannemelijksheidsfunctie zet het gegevensargument (op basis van de geselecteerde variabele) in de theoretische kansmassafunctie vast, en beschouwt deze als een functie van de parameters. De optimalisatieprocedure wordt geïnitialiseerd met de waarden voor de parameters die zijn ingevoerd onder "Toon Verdeling".
 
 ### Uitvoer
-- Schattingen: geeft een tabel weer met de parameterschattingen. Het wijzigen van parametrisering verandert welke parameters worden weergegeven.
-- Soa. error: Geeft de standaardfout van de parameterschattingen weer
-- Betrouwbaarheidsinterval: geeft het betrouwbaarheidsinterval van de parameters weer op het opgegeven betrouwbaarheidsniveau.
+- Schattingen: Geeft een tabel weer met de parameterschattingen. De getoonde parameters passen zich aan aan de gekozen parametrisatie.
+- Std. error: Geeft de standaardfout van de parameterschattingen weer.
+- Betrouwbaarheidsinterval: Geeft het betrouwbaarheidsinterval van de parameters weer op het opgegeven betrouwbaarheidsniveau.
 
 
-## Fit beoordelen
+## Evalueer Passing
 
-### Plots
-- Histogram vs. theoretische pdf: geeft een histogram weer van de geselecteerde variabele overlay met de kansdichtheidsfunctie van de gepaste distributie
-- Empirische vs. theoretische cdf: Geeft een empirische cumulatieve verdelingsgrafiek weer, overlay met de cumulatieve verdelingsfunctie van de gepaste distributie
-- Q-Q-plot: geeft de kwantiel-kwantielgrafiek weer. De * x * -as toont de theoretische kwantielen van de gegevenspunten onder de gepaste verdeling, de * y * -as toont de empirische kwantielen van de geselecteerde variabele.
-- P-P-plot: geeft de kans-waarschijnlijkheidsgrafiek weer. De * x * -as toont de theoretische waarde van de cumulatieve dichtheidsfunctie van de gegevenspunten onder de gepaste verdeling, de * y * -as toont de empirische percentielen van de geselecteerde variabele.
-
-### Statistieken
-- Chi-kwadraat: Toont de chi-kwadraat goedheidstest
+### Grafieken
+- Histogram vs. theoretische pmf (kansmassfunctie): Geeft een histogram weer van de geselecteerde variabele en ook de kansmassafunctie van de gepaste distributie.
```

Comment:
kansmassafunctie

Reply:
Overal 

Histogram vs. theoretische pmf: 

of 

Histogram vs. theoretische pdf: 

van gemaakt. De uitleg staat in de help file.